incident.go

```diff
@@ -874,3 +874,59 @@ func (c *Client) RemoveIncidentNotificationSubscribersWithContext(ctx context.Co
 
 	return &result, nil
 }
+
+type IncidentCustomFieldUpdateOption struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+type IncidentCustomField struct {
+	ID          string `json:"id"`
+	Type        string `json:"type"`
+	Name        string `json:"name"`
+	DisplayName string `json:"display_name"`
+	Description string `json:"description"`
+	DataType    string `json:"data_type"`
+	FieldType   string `json:"field_type"`
+	Value       string `json:"value"`
```

Comment:
This won't work, because sometimes values can be slices for example in the case of a `multi_value_fixed` field type